README.md

```diff
@@ -9,3 +9,19 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+### Proof
+
+A complete graph with $n$ vertices has every pair of vertices connected by an edge. This means the number of edges in each graph is given by $|E| = \binom{n}{2} = \frac{n(n-1)}{2}$. Since $G_1$ and $G_2$ have the same number of vertices and are both completely connected, they also have the same number of edges.
+
+Because the vertex sets of $G_1$ and $G_2$ have the same cardinality ($|V_1| = |V_2| = n$), we can define a bijection $f: V_1 \to V_2$. For any pair of vertices $(u, v) \in E_1$, their images under $f$, $(f(u), f(v))$, are connected in $G_2$ because every vertex in $G_2$ is connected to every other vertex. Similarly, because $f$ is bijective, for every edge $(f(u), f(v)) \in E_2$, the preimages $(u, v) \in E_1$. This ensures that adjacency relationships are preserved under $f$.
```

Comment:
How do you define the bijection? How can you be sure that it has the properties you want?